docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -287,11 +287,13 @@ artifacts cannot be hosted on PyPI because they exceed its size limits.
 
 [Flash-attention](https://github.com/Dao-AILab/flash-attention) does not publish wheels on PyPI at all, but instead
 publishes a customized source distribution that performs platform detection, downloads the appropriate wheel from
-upstream server, and then provides it to the installer. Such an approach can provide a good end-user experience by
-selecting the most optimal variant automatically. However, it prevents `--only-binary` installs from working and
+upstream server, and then provides it to the installer. The [wheel-stub](https://pypi.org/project/wheel-stub/)
+project provides a generic approach to such sdist build redirection techniques.  While these approaches can
+fill this use case, there are several downsides.  Sdist stubs  prevent `--only-binary` installs from working and
```

Comment:
Now you're switching from plural to singular suddenly, this looks wrong. Also I'd avoid saying "sdist", rather "source distribution".